sources/osgGA/FirstPersonManipulator.js

```diff
@@ -205,8 +200,8 @@ define( [
                 if ( Vec2.length( vec ) > 1.0 ) Vec2.normalize( vec, vec );
```

Comment:
why not length2 ?
would even allow Reuse on the normalize replacing with Vec2.mult(vec, 1/length2) ?